tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -1440,7 +1443,7 @@ void StorageManager::decrement_in_progress() {
 }
 
 Status StorageManager::object_remove(const char* path) const {
-  auto uri = URI(path);
+  auto uri = make_uri(path, nullptr);
```

Comment:
again, this and other places below, assign vs contruct (not going to note anymore)